src/linker/ref/Linker.Steps/BaseStep.cs

```diff
@@ -0,0 +1,44 @@
+//
```

Comment:
Use .net foundation header

Reply:
Should the implementation files also have their headers updated?